components/ChallengeQuestionCard.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import '../scss/challengeQuestionCard.scss'
+
+type Props = {
+  title: string
+  question: string
+}
+
+const ChallengeQuestionCard: React.FC<Props> = props => {
+  return (
+    <div className="card border-0">
```

Comment:
Why did you remove `shadow-sm` ?
if you look at Zeplin they use a light shadow without border:
<img width="235" alt="Screen Shot 2020-03-08 at 5 16 09 PM" src="https://user-images.githubusercontent.com/25457563/76173933-2218e400-6161-11ea-98a1-50ed9e846a8d.png">
Card should be `<div className="card shadow-sm border-0">`